gulpfile.js

```diff
@@ -316,6 +316,8 @@ gulp.task('serve-package', () => {
 
 gulp.task('serve-test-pages', () => {
   serveDev(3000, { clientUrl: `//{current_host}:3001/hypothesis` });
+  // Starts an additional dev web server to test cross-origin functionality
+  serveDev(3002, { clientUrl: `//{current_host}:3001/hypothesis` });
```

Comment:
I think this is useful in other context, like for example serving PDFs files from another URL and checking that the fingerprint association allows still to find the annotations.

Reply:
Yes, this seems useful.